workflow/sample_files/base-bldgtype-single-family-attached-2stories.xml

```diff
@@ -183,7 +183,7 @@
             <WallType>
               <WoodStud/>
             </WallType>
-            <Area>587.9</Area>
+            <Area>587.7</Area>

```

Comment:
Still some changes here, which might cause downstream tests (e.g., in OS-ERI) to fail. Can you increase the number of sig fig in the aspect ratio?

Reply:
Ha! Nevermind, you saw it too.